### PR TITLE
Changes requested by IBM Certification

### PR DESCRIPTION
**Pull request by leszko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,39 +52,47 @@ Deploy Hazelcast Enterprise Operator with the following command.
 
     oc apply -f operator.yaml
 
-#### Step 4: Start Hazelcast
+#### Step 4: Create Secret with Hazelcast License Key
 
-Add a Secret within the Project that contains the Hazelcast License Key. If you don't have one, get a trial key from this [link](https://hazelcast.com/hazelcast-enterprise-download/trial/).
+Base encode your Hazelcast License Key. If you don't have one, get a trial key from this [link](https://hazelcast.com/hazelcast-enterprise-download/trial/).
```
Comment (mesutcelik) [inline README.md]: What is Based encode your key? you mean base64?

File: hazelcast.yaml [hunk 2]
```diff
@@ -1,7 +1,361 @@
-apiVersion: hazelcast.com/v1alpha1
+apiVersion: hazelcast.com/v1beta1
```
Comment (mesutcelik) [inline hazelcast.yaml]: beta or just v1?

File: hazelcast.yaml [hunk 3]
```diff
@@ -1,7 +1,361 @@
-apiVersion: hazelcast.com/v1alpha1
```
Comment (mesutcelik) [inline hazelcast.yaml]: What do you think about changing this file into hazelcast-full.yaml and keep existing yaml for easy to start. You can update readme to refer hazelcast-full.yaml if users want to customize their deployment.

File: README.md (hunk 1 above)
Comment (leszko) [reply README.md]: yes, changed, thanks for pointing it!

File: hazelcast.yaml (hunk 2 above)
Comment (leszko) [reply hazelcast.yaml]: I guess we can change it to `v1`

File: hazelcast.yaml (hunk 3 above)
Comment (leszko) [reply hazelcast.yaml]: Good idea!

**Pull request merged**